client/src/main/java/com/paypal/selion/reports/reporter/excelreport/ExcelReport.java

```diff
@@ -30,6 +33,7 @@
 import java.util.Map.Entry;
 import java.util.logging.Level;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
let's stick with `org.apache.commons.lang.StringUtils` unless there is something specific you need in `lang3` here